FitVid/src/Pages/Authentication/ForgotPassword/ForgotPassword.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import Navbar from "../../../Components/Navbar/Navbar";
+import AuthenticationBackground from "../AuthenticationBackground";
+import ForgotPasswordForm from "./ForgotPasswordForm";
+
+function ForgotPassword() {
+  return (
+    <div>
+      <Navbar />
+      <div className="spacer-3"></div>
+      <div className="wrapper">
+        <AuthenticationBackground />
+        <div className="authentication login">
+          <div className="authentication-image-container">
+            <img src="images/fitvid-forgot-password.svg" alt="signup-image" />
```

Comment:
you dont have to mention image in the alt tag as screen readers read it with image added by default